x/stake/keeper/delegation.go

```diff
@@ -24,7 +24,7 @@ func (k Keeper) GetDelegation(ctx sdk.Context,
 	return delegation, true
 }
 
-// return all delegations used during genesis dump
+// return all delegations to a specific validator. Useful for querier.
```

Comment:
shouldn't this one be used for genesis dump and `GetAllDelegationsToValidator ` for querier instead ?

Reply:
Yes, will flip.